docs/source/datasets/index.rst

```diff
@@ -3,30 +3,91 @@
 ========
 Datasets
 ========
-Here we cover the simulated datasets, which are analogous to 'real world' administrative records such as tax documents and census surveys, that users can generate using :code:`pseudopeople` to test their PRL methods. 
+Here we cover the simulated datasets, which are analogous to 'real world' administrative records such as tax documents
+and census surveys, that users can generate using :code:`pseudopeople` to test their PRL methods.
```

Comment:
Let's punch this up---  

`pseudopeople` can general realistic simulated datasets for developing and testing Entity Resolution algorithms and software.

or something

Reply:
Good idea, lets' try to sell it